pkg/irqs/info.go

```diff
@@ -194,8 +194,7 @@ func parseInterrupts(logger *log.Logger, rd io.Reader) (Stats, error) {
 				continue
 			}
 
-			// column 0 is always the IRQ name
-			cpuid := col2cpu[colid-1]
+			cpuid := col2cpu[colid]
```

Comment:
On the first iteration of the ```for``` loop, the value of ```colid``` is 0, and there is no ```cpuid``` on ```col2cpu[0]``` because column 0 is always the IRQ names.
Also you need change the code line [here](https://github.com/openshift-kni/debug-tools/blob/main/pkg/irqs/info.go#L176) to ```col2cpu[colid+1] = cpuid``` for the same reason